jdaviz/configs/imviz/plugins/viewers.py

```diff
@@ -13,7 +13,12 @@
 @viewer_registry("imviz-image-viewer", label="Image 2D (Imviz)")
 class ImvizImageView(BqplotImageView):
 
-    tools = ['bqplot:panzoom', 'bqplot:rectangle', 'bqplot:circle', 'bqplot:matchwcs']
+    # Whether to inherit tools from glue-jupyter automatically. Set this to
+    # False to have full control here over which tools are shown in case new
+    # ones are added in glue-jupyter in future that we don't want here.
+    inherit_tools = False
```

Comment:
@pllim - just FYI this is why the home button was inherited by default before. I think we don't actually want the inheritance here because who knows what tools will get added in glue-jupyter in future so I have disabled it for imviz. We should perhaps do this for other jdaviz apps but this is beyond the scope of this PR.